vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -1228,4 +1231,14 @@ class VectorPreferences @Inject constructor(
         return vectorFeatures.isVoiceBroadcastEnabled() &&
                 defaultPrefs.getBoolean(SETTINGS_LABS_VOICE_BROADCAST_KEY, getDefault(R.bool.settings_labs_enable_voice_broadcast_default))
     }
+
+    fun showIpAddressInDeviceManagerScreens(): Boolean {
```

Comment:
Small question about naming, should we align the methods and preference setting names on `SessionManager` instead of `DeviceManager`?